docs/source/user-guide/tabular/tutorials/plot_quick_train_test_validation.py

```diff
@@ -0,0 +1,199 @@
+# or exported to json using suite_result.to_json()
+suite_result
+
+#%%
+# We can see that we have a problem in the way we've split our data!
+# We've mixed up data from two years, causing a potential leakage.
+# Let's fix this.
+# 
+# Fix Data
+# ^^^^^^^^^^
+
+dt_col = data[datetime_name]
+train_df = data[dt_col.dt.year.isin([2017]) & dt_col.dt.month.isin([6,7,8])]
+test_df = data[dt_col.dt.year.isin([2018]) & dt_col.dt.month.isin([6,7,8])]
+
+#%%
+
+from deepchecks.tabular import Dataset
+
+# Create the new Datasets
+train_ds = Dataset(train_df, **columns_metadata)
+test_ds = Dataset(test_df, **columns_metadata)
+
+#%%
+#
+# Re-run Validation Suite
+# ^^^^^^^^^^^^^^^^^^^^^^^^^
+#
+
+suite_result = validation_suite.run(train_ds, test_ds)
+suite_result
+
+#%%
+#
+# Ok, the date leakage doesn't exist anymore!
+#
+# However, we can see that we have a multivariate drift in the current split, detected by the WholeDatasetDrift check.
+# The drift is detected mainly with a combination of the interest rate (``int_rate``) and loan grade (``sub_grade``).
+#
+# We can consider examining other sampling techniques (e.g. using only data from the same year), ideally achieving one in which the training data's
+# univariate and multivariate distribution is similar to the data on which the model will run (test / production data).
+#
+# If we are planning on training a model with these splits, this drift is worth understanding (is it known? can we do something about it?).
+# Otherwise, we can consider sampling or splitting the data differently, and using deepchecks to validate it.
+
+
+
+
+#%%
+# Run a Single Check
+# -------------------
+#
+# We can run a single check on a dataset, and see the results.
+
+# If we want to run only that check (possible with or without condition)
+from deepchecks.tabular.checks import WholeDatasetDrift
+
+check_with_condition = WholeDatasetDrift().add_condition_overall_drift_value_not_greater_than(0.4)
+# check = WholeDatasetDrift()
+dataset_drift_result = check_with_condition.run(train_ds, test_ds)
+
+#%%
+# We can also inspect and use the result's value:
+
+dataset_drift_result.value
+
+#%%
+# and see if the conditions have passed
+dataset_drift_result.passed_conditions()
+
+#%%
+# Create a Custom Suite
+# ----------------------
+#
+# To create our own suite, we can simply write all of the checks, and add optional conditions.
+
+from deepchecks.tabular import Suite
+from deepchecks.tabular.checks import TrainTestFeatureDrift, WholeDatasetDrift, \
+ TrainTestPredictionDrift, TrainTestLabelDrift
+
+drift_suite = Suite('drift suite',
+TrainTestFeatureDrift().add_condition_drift_score_not_greater_than(
+  max_allowed_categorical_score=0.2, max_allowed_numeric_score=0.1),
+WholeDatasetDrift().add_condition_overall_drift_value_not_greater_than(0.4),
+TrainTestLabelDrift(),
+TrainTestPredictionDrift()
+)
+
+#%%
+#
+# we can run our new suite using:
+
+result = drift_suite.run(train_ds, test_ds)
```

Comment:
why here and not in a different guide? Also, if it's here we better have model evaluation checks in this custom suite.